src/status/chat.nim

```diff
@@ -127,6 +129,30 @@ proc updateContacts*(self: ChatModel, contacts: seq[Profile]) =
 
 proc init*(self: ChatModel, pubKey: string) =
   var chatList = status_chat.loadChats()
+  var contacts = getAddedContacts()
+
+  let profileUpdatesChatIds = chatList.filter(c => c.chatType == ChatType.Profile).map(c => c.id)
+
+  if chatList.filter(c => c.chatType == ChatType.Timeline).len == 0:
+    var timelineChannel = newChat(status_utils.getTimelineChatId(), ChatType.Timeline)
+    self.join(timelineChannel.id, timelineChannel.chatType)
+    chatList.add(timelineChannel)
+
+  if not profileUpdatesChatIds.contains("@" & pubKey):
+    var profileUpdateChannel = newChat("@" & pubKey, ChatType.Profile)
+    status_chat.saveChat(profileUpdateChannel.id, profileUpdateChannel.chatType, true, "#000000", "", pubKey)
```

Comment:
Consider changing the definition of saveChat to:
```
proc saveChat*(chatId: string, oneToOne: bool = false, active: bool = true, color: string = "#000000", ensName: string = "")
```

then, replace this call by
```
status_chat.saveChat(profileUpdateChannel.id, profileUpdateChannel.chatType, profile=pubKey)
```

Reply:
You want me to remove the `profile` argument and then pass it down explicitly?
I don't think that's gonna compile?